docs/content/operator/common/archives.mdx

```diff
@@ -95,24 +106,32 @@ state-archive-read-config:
 </Tabs>
 
 
-Even though these buckets are publicly readable, you need to make sure to properly grant the correct policies to read them via AWS, for example:
-```
+### Configure AWS IAM Permissions for Archive Access
```

Comment:
All the information about AWS S3 should be commented out for now, since we currently do not have a public AWS S3 bucket yet. It is better to comment out the information for now.

Right now we only have a Hetzner S3 public state sync archive and are looking into other options.